portal-frontend/src/app/features/applications/application-details/parcel/parcel.component.html

```diff
@@ -148,7 +148,6 @@ <h5 *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">Government
 
       <div
```

Comment:
Need to clean up all the conditions and fields from here to line 186 as well

Reply:
Not all of them are cleaned up yet, see landowner, organization, corporate summary, etc.

Reply:
removed

Reply:
Line 168 still needs to be addressed